### PR TITLE
Updates challenge to return boolean expression

### LINKED ISSUE #24026: Object Oriented Programming: Understand the Constructor Property returns boolean values in example
**Describe your problem and how to reproduce it:**
Following the discussion in [#18925](https://github.com/freeCodeCamp/freeCodeCamp/issues/18925), here is another instance of an example (`joinBirdFraternity`) returning boolean values instead of using a boolean expression. Do we want this to be brought in line similar to the changes being made there?

**Add a Link to the page with the problem:**
https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/object-oriented-programming/understand-the-constructor-property

### PR DESCRIPTION
**Pull request by tfantina**
Based on [#24026](https://github.com/freeCodeCamp/freeCodeCamp/issues/24026#issuecomment-431666277) I've changed this challenge to return a boolean expression rather than boolean values.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the 24026 below with the issue number.

Closes #24026

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/understand-the-constructor-property.english.md [hunk 1]
```diff
@@ -67,7 +67,11 @@ function Dog(name) {
   this.name = name;
 }
 function joinDogFraternity(candidate) {
-  return candidate.constructor === Dog;
```
Comment (ValeraS) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/understand-the-constructor-property.english.md]: Here is returned a boolean expression, but You return strings.

**Comment by ValeraS** [comment]
@RandellDawson IMO we should close it.

**Comment by RandellDawson** [comment]
I agree.

**Comment by RandellDawson** [comment]
Thank you for submitting a pull request to attempt to resolve PR#24026.  However, upon review, your additions change the function from returning a Boolean value, which is what the original challenge did, to returning strings.  In the original issue PR #24026, it was asked whether or not to return a Boolean expression or explicitly return `true` and `false`.  You completely changed the challenge to return strings which is unnecessary.  

The focus of this challenge is on understanding the Constructor Property and not on how to return Boolean expressions.   The only suggested change which would fit with @mail-liam's original suggestion would be to just change the example in the Description section to return a Boolean expression.  This way, it makes the code concise and does not involve needing to change the Tests section of the challenge.  If you want to do that, then please create a new PR with just that change.  

Thank you.

**Pull request closed without merging**